### PR TITLE
Bump cert-manager versions to 1.13 (as described in the release-process)

### PR DESCRIPTION
**Pull request by inteon**
see https://cert-manager.io/docs/contributing/release-process/

### CODE REVIEW

File: content/v1.13-docs/contributing/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+title: Contributing
+description: 'cert-manager contributing guide: Get involved!'
+---
+
+## Great to See You!
+
```
Comment (SgtCoDFish) [inline content/v1.13-docs/contributing/README.md:7]: suggestion: we don't usually include the "contributing" section for older release docs

File: content/docs/installation/supported-releases.md [hunk 2]
```diff
@@ -18,23 +18,22 @@ and other world events.
 
 | Release  | Release Date |      End of Life       | [Supported Kubernetes versions][s] | [Supported OpenShift versions][s] |
 |----------|:------------:|:----------------------:|:----------------------------------:|:---------------------------------:|
-| [1.12][] | May 19, 2023 |    Release of 1.14     |            1.22 → 1.27             |            4.9 → 4.14             |
-| [1.11][] | Jan 11, 2023 |    Release of 1.13     |            1.21 → 1.27             |            4.8 → 4.14             |
```
[the remark this answers is not in the document]
Comment (inteon) [reply content/docs/installation/supported-releases.md:22]: I added a note, PTAL.

**Review by SgtCoDFish — APPROVED** [review]
/lgtm
/approve

Thanks for doing this, it's not a fun job but someone's gotta do it!

**Pull request merged**